blocks/contact-form/submit-contact-info.js

```diff
@@ -0,0 +1,40 @@
+export default function submitContactInfo(inputs) {
+  // Extracting values from input fields
+  const firstName = inputs[0].value;
+  const lastName = inputs[1].value;
+  const email = inputs[2].value;
+  const dateOfBirth = inputs[3].value;
+  const agreeToTos = inputs[4].checked;
```

Comment:
This data map is still directly tied to the initial structure of the block. If an author changes the order of the fields, this mapping breaks.
Try relying on available metadata of the input fields (if necessary , adding that metadata during `decorate` yourself) instead of assuming an order of values.